### PR TITLE
HPCC-32531 Document BareMetal Logaccess config

### PR DESCRIPTION
**Pull request by rpastrana**
- Adds baremetal section to logaccess README
- Adds new xml sample fle

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by rpastrana** [comment]
@g-pan please look over README file changes

File: helm/managed/logging/loki-stack/README.md [hunk 1]
```diff
@@ -200,3 +200,24 @@ Example use:
       logFormat:
         type: "json"
 ```
+#### Baremetal configuration
+
+LogAccess can be configured on baremetal systems as well. It is not a common scenario because baremetal logs continue to be persisted on the local filesystem. Enabling LogAccess on a baremetal system would allow users to fetch remotely stored HPCC logs (not likely related to the host HPCC Systems deployment). To enable, inject a logAccess block with all configuration values properly filled in. The can be injected within the Sofware element in the active environment.xml or within the global element on the esp.xml.
```
Comment (g-pan) [inline helm/managed/logging/loki-stack/README.md:205]: The can be injected  -- 
s/b "This" or possibly "The block"

**Review by g-pan — COMMENTED** [review]
one minor comment

File: helm/managed/logging/loki-stack/README.md (hunk 1 above)
Comment (g-pan) [reply helm/managed/logging/loki-stack/README.md:205]: also Sofware s/b software

**Comment by rpastrana** [comment]
thanks @g-pan. Changes made and squashed, please confirm.

**Review by g-pan — APPROVED** [review]
approved

**Comment by rpastrana** [comment]
@ghalliday please merge

**Pull request merged**